.travis.yml

```diff
@@ -1,15 +1,20 @@
 language: python
+
 python:
   - "3.6"
 
 cache: pip
 
+env:
+  matrix:
+    - TOX_ENV=python
+    - TOX_ENV=flake8
+
```

Comment:
Here's where we define each new environment for Travis to run.  For now, this is really just setting the value of an environment variable.